components/config/definition.rst

```diff
@@ -425,9 +425,34 @@ All options can be documented using the
 :method:`Symfony\\Component\\Config\\Definition\\Builder\\NodeDefinition::info`
 method.
 
+.. code-block:: php
+
+    $rootNode
+        ->children()
+            ->inetegerNode('entries_per_page')
```

Comment:
This clearly proves that I didn't review anything when creating this PR. It's fixed now. Thanks.